styles/shop.css

```diff
@@ -77,3 +77,24 @@ input[type="radio"]{
     background: #04AA6D;
     cursor: pointer;
   }
+  @media only screen and (max-width:768px){
+    main{
+        grid-template-areas:  "aside aside aside aside"
+        "shop shop shop shop";
+    }
+    .filters-wrapper{
+        position:relative;
+    }
+    .filter-body-wrapper{
+        position:absolute;
+        z-index:1;
+        width:100%;
+        background-color: var(--bg-secondary);
+        top:-40rem;
+        transition: all 500ms linear;
```

Comment:
It is better to use `transform: translateY(-40rem)` instead of `top: -40rem`. I was doing the same thing until I was told not to in a PR review today.
Heres why: https://www.paulirish.com/2012/why-moving-elements-with-translate-is-better-than-posabs-topleft/

Reply:
Changed it. Thank you for letting me know about this :)